docs/ssh.md

```diff
@@ -0,0 +1,141 @@
+# SSH into a VirtualMachineInstance
```

Comment:
this looks like user-guide material. what's the plan here for the user-guide?

Reply:
No plans as of yet, but I agree it should be in there.